{
  "id": 1392,
  "name": "ERROR_CLUSTER_DATABASE_TRANSACTION_NOT_IN_PROGRESS",
  "description": "An internal cluster error occurred. There was an attempt to commit a cluster database transaction while no transaction was in progress.",
  "external_url": "https://github.oblokos.com/?art=ERROR_CLUSTER_DATABASE_TRANSACTION_NOT_IN_PROGRESS",
  "image": "https://github.oblokos.com/art/ERROR_CLUSTER_DATABASE_TRANSACTION_NOT_IN_PROGRESS.png",
  "opensea_url": "https://opensea.io/assets/matic/0x252f9390feb63aeb00042dd5c4ff84af33f42b85/1392",
  "attributes": [
    {
      "trait_type": "artist",
      "value": "0x5Ae65e30834f61B26A25D9F9F0847c3da11dD391"
    },
    {
      "trait_type": "collection",
      "value": "Metaverse First Contact"
    },
    {
      "trait_type": "error_code",
      "value": "5919 (0x171F)"
    },
    {
      "display_type": "number",
      "trait_type": "frame",
      "value": "28815"
    },
    {
      "display_type": "number",
      "trait_type": "index",
      "value": 0.606
    },
    {
      "display_type": "number",
      "trait_type": "level",
      "value": 0
    }
  ]
}